.gitignore

```diff
@@ -275,3 +275,7 @@ GitHub.sublime-settings
 
 *hmr.js
 .vscode/settings.json
+
+dash_vtk
+deps
```

Comment:
This is needed to generate the r files to build.

Reply:
But should we push all new files each time we build? Or is it just a build-tree that will be filled up in the ci but does not needs to be commited.

Reply:
I usually push two commits, one where I make some changes to the source code and the second one is simply the output of `npm run build`